crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1938,13 +1941,20 @@ impl SymbolicationActor {
                     ),
                     |(cfi_caches, minidump_path, spawn_time, return_modules)| {
                         let procspawn::serde::Json(cfi_caches) = cfi_caches;
-                        stackwalk_with_rust_minidump(
-                            cfi_caches,
-                            minidump_path,
-                            spawn_time,
-                            return_modules,
-                        )
-                        .map(procspawn::serde::Json)
+                        let rt = tokio::runtime::Builder::new_current_thread()
+                            .enable_all()
+                            .build()
+                            .unwrap();
+                        rt.block_on(async move {
+                            stackwalk_with_rust_minidump(
+                                cfi_caches,
+                                minidump_path,
+                                spawn_time,
+                                return_modules,
+                            )
+                            .await
+                            .map(procspawn::serde::Json)
+                        })
```

Comment:
```suggestion
                        procspawn::serde::Json(rt.block_on(
                            stackwalk_with_rust_minidump(
                                cfi_caches,
                                minidump_path,
                                spawn_time,
                                return_modules,
                            )
                        ))
```
(+ reformatting ;-)

Might be a bit simpler (and avoid un unneeded async move block)